oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/RelatedSelectField/index.js

```diff
@@ -1 +1,4 @@
-export { RelatedSelectField } from "./RelatedSelectField";
+export {
+  RelatedSelectField,
+  serializeVocabularySuggestions,
```

Comment:
Shloud by named just `serializeSuggestions`

Reply:
The issue is that serializeSuggestions also exists in this module and is default serializer. If someone wants to use vocabularies serializer, then I thought it would make sense to keep it in the same module and enable it as an export to be used as some predefined serializer that component user could use. I don't know where else to put it. And finally, if you don't wish to use default or vocabulary serializer you can write and provide your own serializer as a prop to the component.

Reply:
The point is, that if someone wants to use vocabularies serializer, it would't work for him without also installing `oarepo-vocabularies` library. So I think that this one should be a part of `@js/oarepo_vocabularies`